logic/addons.py

```diff
@@ -0,0 +1,25 @@
+from logic.base_logic import Base_Logic
+from logic.neuro_feedback import Neuro_Feedback
+from utils import map2dto1d
+
+class Addons(Base_Logic):
+    def __init__(self, board, logic_name="addons", window_seconds=2, normalize_scale=1.1, ema_decay=0.025):
```

Comment:
I did some more looking around and it turns out I can get the type of an object and then turn it into a string, it would be better than just getting the logic name manually like this. I'll give that a shot if that is never supposed to change